azure-iot-hub/tests/test_iothub_http_runtime_manager.py

```diff
@@ -0,0 +1,77 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import pytest
+from azure.iot.hub.protocol.models import AuthenticationMechanism
+from azure.iot.hub.iothub_http_runtime_manager import IoTHubHttpRuntimeManager
+
+"""---Constants---"""
+
+fake_hostname = "beauxbatons.academy-net"
+fake_device_id = "MyPensieve"
+fake_shared_access_key_name = "alohomora"
+fake_shared_access_key = "Zm9vYmFy"
+fake_lock_token = "fake_lock_token"
+
+
+"""----Shared fixtures----"""
+
+
+@pytest.fixture(scope="function", autouse=True)
+def mock_http_runtime_operations(mocker):
+    mock_http_runtime_operations_init = mocker.patch(
+        "azure.iot.hub.protocol.iot_hub_gateway_service_ap_is.HttpRuntimeOperations"
+    )
+    return mock_http_runtime_operations_init.return_value
+
+
+@pytest.fixture(scope="function")
+def iothub_http_runtime_manager():
+    connection_string = "HostName={hostname};DeviceId={device_id};SharedAccessKeyName={skn};SharedAccessKey={sk}".format(
+        hostname=fake_hostname,
+        device_id=fake_device_id,
+        skn=fake_shared_access_key_name,
+        sk=fake_shared_access_key,
+    )
+    iothub_http_runtime_manager = IoTHubHttpRuntimeManager(connection_string)
+    return iothub_http_runtime_manager
+
+
+@pytest.mark.describe("IoTHubHttpRuntimeManager - .receive_feedback_notification()")
+class TestReceiveFeedbackNotification(object):
+    @pytest.mark.it("Receive feedback notification")
```

Comment:
>Receive feedback notification [](start = 21, length = 29)

Here and everywhere, more descriptive test descriptions, in the grammatical format as if the sentence has started with "It"

e.g. "Uses protocol layer HTTP runtime to receive feedback notifications"